contracts/staking/StakingMessages.sol

```diff
@@ -207,6 +310,37 @@ library StakingMessages {
      * +--------------+----------+----------+
      *                           | 44 bytes |
      *                           +----------+
+     *
+     * @param validationID The ID of the validation period.
+     * @param uptime The uptime of the validator.
+     * @return The packed message.
+     */
+    function packValidationUptimeMessage(
```

Comment:
does `abi.encodePacked` work for these packs? ref: https://github.com/ava-labs/teleporter/pull/482